tests/health_mgmtapi/health-cmdapi-test.sh.in

```diff
@@ -0,0 +1,263 @@
+	if [ $? -ne 0 ] ; then
+		echo -e "${RED}ERROR: Unexpected response '$resp'"
+		err=$((err+1))
+	elif [ "${r}" != "${2}" ] ; then
+		echo -e "${RED}ERROR: 'Got ${r}'. Expected '${2}'"
+		err=$((err+1))
+	else
+		echo -e "${GREEN}Success"
+	fi
+}
+
+cmd () {
+	echo -e "${WHITE}Cmd '${1}', expecting '${2}'"
+	RESPONSE=$(curl -s "http://$URL/api/v1/manage/health?${1}" -H "Authorization: Bearer $TOKEN" 2>&1)
+	if [ "${RESPONSE}" != "${2}" ] ; then
+		echo -e "${RED}ERROR: Response '${RESPONSE}' != '${2}'"
+		err=$((err+1))
+	else
+		echo -e "${GREEN}Success"
+	fi
+}
+
+WHITE='\033[0;37m'
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+GRAY='\033[0;37m'
+
+SETUP=0
+RESTART=0
+CLEANUP=0
+TEST=0
+URL="localhost:19999"
+
+while getopts :srctu: option
+do
+    case "$option" in
+    s)
+        SETUP=1
+        ;;
+	r)
+		RESTART=1
+		;;
+	c)
+		CLEANUP=1
+		;;
+	t)
+		TEST=1
+		;;
+	u)
+		URL=$OPTARG
+		;;
+	*)
+		printhelp
+		exit 1
+		;;
+	esac
+done
+
+if [ $SETUP -eq 1 ] ; then
+	echo "Preparing netdata configuration for testing"
+	# Prep netdata for tests
+	if [ -f "${NETDATA_USER_CONFIG_DIR}/python.d.conf" ] ; then
+		cp -f "${NETDATA_USER_CONFIG_DIR}/python.d.conf"  /tmp/python.d.conf
+	else
+		cp "${NETDATA_STOCK_CONFIG_DIR}/python.d.conf" "${NETDATA_USER_CONFIG_DIR}/"
+	fi
+	sed -i -e "s/example: no/example: yes/g" "${NETDATA_USER_CONFIG_DIR}/python.d.conf"
+
+	mypath=$(cd ${0%/*} && echo $PWD)
+
+	cp -f "${mypath}/python-example.conf" "${NETDATA_USER_CONFIG_DIR}/health.d/"
+
+	# netdata.conf
+	if [ -f "${NETDATA_USER_CONFIG_DIR}/netdata.conf" ] ; then
+		cp -f "${NETDATA_USER_CONFIG_DIR}/netdata.conf"  /tmp/netdata.conf
+	fi
+	printf "[health]\nrun at least every seconds = 1\n" > "${NETDATA_USER_CONFIG_DIR}/netdata.conf"
+
+	chmod +r "${NETDATA_USER_CONFIG_DIR}/python.d.conf" "${NETDATA_USER_CONFIG_DIR}/netdata.conf" "${NETDATA_USER_CONFIG_DIR}/health.d/python-example.conf" "${NETDATA_STOCK_CONFIG_DIR}/health.d/load.conf"
+	# Restart netdata
+	if [ $RESTART -eq 1 ] ; then
+		echo "Restarting netdata"
+		systemctl restart netdata
+	fi
+fi
+
+err=0
+
+# Execute tests
+if [ $TEST -eq 1 ] ; then
+
+	HEALTH_CMDAPI_MSG_AUTHERROR="Auth Error"
+	HEALTH_CMDAPI_MSG_SILENCEALL="All alarm notifications are silenced"
+	HEALTH_CMDAPI_MSG_DISABLEALL="All health checks are disabled"
+	HEALTH_CMDAPI_MSG_RESET="All health checks and notifications are enabled"
+	HEALTH_CMDAPI_MSG_DISABLE="Health checks disabled for alarms matching the selectors"
+	HEALTH_CMDAPI_MSG_SILENCE="Alarm notifications silenced for alarms matching the selectors"
+	HEALTH_CMDAPI_MSG_ADDED="Alarm selector added"
+	HEALTH_CMDAPI_MSG_INVALID_KEY="Invalid key. Ignoring it."
+	HEALTH_CMDAPI_MSG_STYPEWARNING="WARNING: Added alarm selector to silence/disable alarms without a SILENCE or DISABLE command."
+	HEALTH_CMDAPI_MSG_NOSELECTORWARNING="WARNING: SILENCE or DISABLE command is ineffective without defining any alarm selectors."
+
+	if [ -f "${NETDATA_REGISTRY_DIR}/../netdata.api.key" ] ;then
```

Comment:
Why this is referenced in context of `registrydir` directory and not `varlibdir`?

Reply:
I didn't see valribdir in subst.inc, will fix it in another PR

Reply:
there is also no `registrydir` since this PR is adding it, look here: https://github.com/netdata/netdata/pull/4969/files#diff-8effaf1d5895dc974fa6deac404062d0R8

Reply:
Added to subst.inc and replaced.